core/settings/qPref.h

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-2.0
+#ifndef QPREF_H
+#define QPREF_H
+
+#include <QObject>
+#include "core/pref.h"
+
+class qPref : public QObject {
+	Q_OBJECT
+	Q_ENUMS(cloud_status);
+
+public:
+	qPref(QObject *parent = NULL) : QObject(parent) {};
```

Comment:
Any reason to inline the constructor?
(Note: If you do, then the semicolon is not needed.)

Reply:
The reason was that if I did not make it inline, I got a compile error if it was not declared in the cpp file. But maybe I made something wrong ?

I am happy the remove the {}; if I do not have to declare it in the cpp file.

Reply:
> The reason was that if I did not make it inline, I got a compile error if it was not declared in the cpp file. 

Of course - if you don't inline, you have to define it somewhere.

> I am happy the remove the {}; if I do not have to declare it in the cpp file.

Not sure that I understand.
1) We define the constructor of every other class in the .cpp file, so why would this be a problem here?
2) You don't pass a parent object anyway, so you can completely remove the declaration and just use the default constructor.

The destructor can also be removed. The default-constructor is good enough.

Note: If you have `{}`, then you don't need `;':
```
	qPref(QObject *parent = NULL) : QObject(parent) {}
```